src/containers/Form.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { EntityDataModelApi, DataApi } from 'lattice';
+
+import FormView from '../components/FormView';
+
+class Form extends React.Component {
+	constructor(props) {
+		super(props);
+
+		this.state = {
```

Comment:
It would be better to extract this into an `INITIAL_STATE` object that you can reference, similar to how we do it with redux in reducers.